README.md

```diff
@@ -1,12 +1,40 @@
 # Maestro
+
 Maestro is a system to leverage [CloudEvents](https://cloudevents.io/) to transport Kubernetes resources to the target clusters, and then transport the resource status back. The resources are stored in a database and the status is updated in the database as well. The system is composed of two parts: the Maestro server and the Maestro agent.
 - The Maestro server is responsible for storing the resources and their status. And sending the resources to the message broker in the CloudEvents format. The Maestro server provides Resful APIs and gRPC APIs to manage the resources.
 - Maestro agent is responsible for receiving the resources and applying them to the target clusters. And reporting back the status of the resources.
 
 ## Architecture
+
 Taking MQTT as an example:
+
 ![Maestro Architecture](./arch.png)
 
+## Why
+
+Maestro was created to address several critical challenges associated with managing Kubernetes clusters at scale.
```

Comment:
This paragraph is good in that we should see performance and scalability improvements by using Maestro, as you note.

Another motivation (I think, perhaps, primary motivation) was to improve the security architecture.

By using Maestro, we do not need kubeconfigs.  Our Cluster Service (CS) deployment would no longer need kubeconfigs or direct access to customer clusters. CS just uses the Maestro API. 

Only the Maestro consumer in a management cluster needs kubeconfigs, thus improving security.

Reply:
Thanks Mark for the review!

Could you please help me word it in a way that's more generic and not mention CS, SD, MC, HCP etc?
I am trying to show maestro as a possible solution for another project and it's not related to SD. Maybe closer to self managed?

Reply:
Added a generic sentence about not needing kubeconfig. Maybe it's a bit vague but PTAL. Thanks.